README.md

```diff
@@ -1,10 +1,34 @@
+![badge-img](https://img.shields.io/badge/Made%20at-%23AstroHackWeek-8063d5.svg?style=flat)
 # stateoftheuniverse
-Returns the state of the universe
-An Astro Hack Week 2019 project
+## A python package that returns the current state of the universe
 
+Would you like to know when the phase of the moon? What about the current tally of exoplanets and asteroids? Curious about the constellations and planets you can see in the sky tonight? Want to celebrate the birthdays of historical astronomers?
+
+Then you are going to love stateoftheuniverse.
+
+Input your location, and we'll do the rest. 
```

Comment:
can you add a little section on how to install? Just like
```
pip install stateoftheuniverse --user
```

Reply:
Actually at the moment teach them how to download it and install, so

```
git clone https://github.com/ojhall94/stateoftheuniverse
cd stateoftheuniverse
python3 setup.py install --user
```

Reply:
Got it, thanks!